tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -158,28 +163,32 @@ describe(__filename, () => {
     expect(link).toHaveProp('href', 'mailto:ba@bar.com');
   });
 
-  it('renders the version number of an add-on', () => {
-    const addon = createInternalAddon({
-      ...fakeAddon,
-      current_version: {
-        ...fakeAddon.current_version,
-        version: '2.0.1',
-      },
+  const _loadVersions = (versionProps = {}) => {
+    return loadVersions({
```

Comment:
Usually we just include `store.dispatch()` in the helper itself since that's what you always need to do.